docs/versioned_docs/version-7.1.x/configuration/auth.md

```diff
@@ -300,9 +300,10 @@ you may wish to configure an authorization server for each application. Otherwis
     skip_provider_button = true
     ```
 
-The `oidc_issuer_url` is based on URL from your **Authorization Server**'s **Issuer** field in step 2, or simply https://corp.okta.com .
+The `oidc_issuer_url` is based on URL from your **Authorization Server**'s **Issuer** field in step 2, or simply https://corp.okta.com (If your not using Authorization Server).
```

Comment:
What about this instead, it's a bit more explicit?
```suggestion
The `oidc_issuer_url` is based on the URL from your **Authorization Server**'s **Issuer** field in step 2, or if you are not using an Authorization server, the global issuer URL https://corp.okta.com.
```

Reply:
ok looks good. not sure how to accept your change's
and update the main docs

Reply:
I think you can just click commit suggestion to accept the change, or copy paste and overwrite manually

And the main docs would be in https://github.com/oauth2-proxy/oauth2-proxy/blob/master/docs/docs/configuration/auth.md, it should look pretty much identical